etc/cdn-log-shipper/log-shipper.yml

```diff
@@ -146,6 +157,12 @@ Resources:
             // podcast id and episode guid (only works for dovetail3-cdn requests)
```

Comment:
Not related directly to this change but ...

For re-processing purposes, it may be useful to also have this lambda log what S3 input file it's processing, and how many rows it had.  Just above this line somewhere:

```js
console.info(`Read ${rows.length} rows from s3://${Bucket}/${Key}`);
```